### DISCUSSION TITLE
Web UI Feature Request

### DISCUSSION THREAD

**Discussion by spatel7786** [jgyates/genmon#1053] [Ideas]
@jgyates  - Taking a shot in the dark here and seeing a certain feature can be added under the maintenance section. I would like to see if a stand alone toggle can be added to turn "Quiet Mode" on or off during exercise.  I have an evolution 2.0 controller Generac.

**Comment by jgyates** [comment]
are you asking to change the quite mode setting while an exercise is taking place or are you asking to have a separate control for quite mode? 

I would need more info one the reasoning for a separate control as this code has been in place for some time and I do not want to change it without a good reason. If you let me know more about your goal I may be able to offer alternate solutions.

Changing the mode during an exercise cycle will not change the current running cycle mode.

**Reply by spatel7786** [reply]
@jgyates 
Asking for a separate control. Just wanted to see if a toggle can be added that will let me control the quiet mode function from the comforts of my home. There are times I would like the exercise to run at full operating speeds rather than half the RPM. Currently I have to go out to the unit and turn quiet mode off prior to the exercise time and then remember to turn it back on later in the month. Reason being is I live next to people who travel for work every other month so I try not to annoy them with the generator exercising at full speed when they are home. When they are out of town, I let the generator run at full during weekly exercise.

**Reply by jgyates** [reply]
ok, sorry, I understand now. When originally asked the question I thought you wanted the existing control moved. I forgot that quiet mode was only available on some models until I went and looked at the original code. This issue is that only Evolution Liquid Cooled models have the ability to write quite mode over modbus. So your requeist has already been implimented, but only for controllers that support this functionality over modbus. When you originally asked the questions I was assuming that it was already available on all models (I have a liquid cooled Evo so it is the one I see running the most).

Here is the Evo Liquid Cooled maint page:

<img width="649" alt="Screenshot 2024-02-14 at 8 55 56 PM" src="https://github.com/jgyates/genmon/assets/4513286/f978fd55-6ee1-442b-b1b9-554115069066">

**Reply by spatel7786** [reply]
Ah shame on me for not reading! Sorry to waste your time on this as it was clearly already covered and implemented for units that can handle this. My apologies. Never the less still an AMAZING program!

**Comment by jgyates** [comment]
.
